lib/manager/npm/post-update/index.ts

```diff
@@ -441,6 +441,10 @@ export async function getAdditionalFiles(
       './others/yarn',
       'YARN_CACHE_FOLDER'
     ),
+    YARN_GLOBAL_FOLDER: await ensureCacheDir(
+      './others/berry',
+      'YARN_GLOBAL_FOLDER'
+    ),
```

Comment:
@rarkins This need to be adapted in v26

Reply:
AFAIS this wouldn't be a breaking change.

Reply:
Was just a note for our v26 branch. 😅